style.css

```diff
@@ -193,79 +223,48 @@ header {
   margin-bottom: 5%;
 }
 
-.intro {
-  margin: 0;
-  max-width: 37.5rem;
-  padding: 7% 7%;
-  background: url('./images/header-shapes-mobile.svg') 0 50px no-repeat;
-  background-size: 100% auto;
-  height: 750px;
-}
-
 .sublinks > ul {
   padding: 0;
   display: flex;
-  width: 70%;
+  width: 60%;
   justify-content: space-around;
   flex-direction: row;
   list-style-type: disc;
-  color: grey;
+  color: #ebebff;
+  margin-left: 5%;
+  margin-bottom: 2em;
 }
 
 .sublinks > ul > li {
   list-style-type: disc;
+  position: relative;
+  left: -3px;
 }
 
 .sublinksround > ul {
   padding: 0;
   display: flex;
-  width: 70%;
   justify-content: space-around;
+  align-items: left;
   flex-direction: row;
   list-style-type: disc;
   margin-top: 5%;
-  margin-bottom: 10%;
-  color: #6070ff;
+  margin-bottom: 1%;
 }
```

Comment:
- [ ] Great job 👏 fixing the width issue, but you seem to be missing one thing according to the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0-1&t=T8hK6peKRbaE8o8E-0),
    - Figma design:
      ![firefox_IsUjFneqXY](https://user-images.githubusercontent.com/104968236/229070232-fcbfff43-18fd-4d4d-9e66-aa43e6f0b856.png)
    - As I've highlighted in yellow above there's some right padding in the container of the list of technologies so please make sure to add that as this will ensure to your future clients you can follow a given design to perfection.🎯
    -  Your design:
      ![firefox_Z2tB8A42aY](https://user-images.githubusercontent.com/104968236/229070606-7ea5220a-860a-41de-b2da-2834eb510734.png)